docs/presentations/2024-08-27-CU-Anschutz-DBMI-Retreat/readme.md

```diff
@@ -0,0 +1,45 @@
+# 2024-08-27 University of Colorado Anschutz Medical Campus - DBMI Retreat Poster
+
+The content here is for creating a poster for the University of Colorado Anschutz Medical Campus - Department of Biomedical Informatics (DBMI) Annual Retreat poster session.
```

Comment:
Do we want to include SBI2 here also?

Reply:
Thanks good question - I wasn't sure what might happen in terms of SBI2 so provided specific information about what we know at the moment.